packages/constants/src/__test__/logLevels.test.js

```diff
@@ -0,0 +1,16 @@
+import { logLevels } from '..'
+
+describe('logLevels', () => {
+	describe('entries', () => {
+		it('should be camelCase', () => {
+			Object.keys(logLevels).forEach((logLevel) => {
+				expect(logLevel).toMatch(/^[a-z][a-zA-Z]*/)
```

Comment:
This needs the end-of-string anchor, or else snake-cased stuff gets through. Test `export const fart_ing = 'fart_ing'` if you want to see. (On that note, that may mean the matcher itself should be a utility method so you can test that _it_ is working as expected through both good and failing cases, but I leave that up to you.)

```suggestion
				expect(logLevel).toMatch(/^[a-z][a-zA-Z]*$/)
```